bin/build-locales.js

```diff
@@ -0,0 +1,18 @@
+#! /usr/bin/env node
+
+const spawn = require('cross-spawn');
+
+let locales = '*';
+if (process.env.NODE_ENV === 'production' &&
+    process.env.npm_package_config_productionLocales) {
+  locales = process.env.npm_package_config_productionLocales;
+}
+
+spawn.sync('pontoon-to-webext', [], {
+  stdio: 'inherit',
+  env: Object.assign(
+    {},
+    process.env,
+    { SUPPORTED_LOCALES: locales }
+  )
+});
```

Comment:
Should we get pontoon-to-webext to just check a key in the npm config (if it exists), so that we could remove this extra file?
…
I guess we would still want to do it only if it was for production, and that doesn't seem like something that pontoon-to-webext would really be interested in, so having this file seems fine.